### PR TITLE
(BIDS-1746) Add Withdrawal webhook option

### PR DESCRIPTION
**Pull request by Eisei24**

### CODE REVIEW

**Comment by LuccaBitfly** [comment]
The BIDS description says to remove the withdrawal checkbox from the `/user/notifications` view but it is still there.

**Comment by LuccaBitfly** [comment]
Addendum to my last comment: the BIDS description also says to disable this feature for e-mail notifications. Am I correct in assuming that you did not disable the possibility e-mail notifications to include withdrawals? Regardless of whether the checkbox is available for the user to change.

**Review by LuccaBitfly — COMMENTED** [review]
pls ignore this comment

File: handlers/user.go [hunk 1]
```diff
@@ -2455,7 +2455,7 @@ func NotificationWebhookPage(w http.ResponseWriter, r *http.Request) {
```
Comment (LuccaBitfly) [inline handlers/user.go:2455]: The BIDS description says to remove the withdrawal checkbox from the /user/notifications view but it is still there. The description also says to disable the feature for e-mail notifications. Am I correct in assuming that you did not remove the possibility for e-mails to include withdrawals? Irregardless of whether the checkbox is available for the user to change.

Comment (Eisei24) [reply handlers/user.go:2455]: Right, I at first removed the checkbox but then had to revert is again, see the last comment in the BIDS.

You currently can't disable email notifications for specific notifications just for all in general.
And the validators set for those email notifications are then used for the webhooks.

So we decided to only add the webhooks option for now.

**Review by LuccaBitfly — APPROVED** [review]
lgtm

**Pull request merged**